### PR TITLE
Removing Opensource.gz reference

### PR DESCRIPTION
**Pull request by finchy**
We're going to cease distribution of this file.

### CODE REVIEW

**Comment by finchy** [comment]
Now, I would suggest that we remove all the processing code to do this as well.

File: pulledpork.pl [hunk 1]
```diff
@@ -191,7 +191,6 @@ sub Help {
    -p <path to your snort binary> -C <path to your snort.conf> -t <sostub output path>
    -h <changelog path> -H <signal_name> -I (security|connectivity|balanced) -i <path to disablesid.conf>
    -b <path to dropsid.conf> -e <path to enablesid.conf> -M <path to modifysid.conf>
-   -r <path to docs folder> -K <directory for separate rules files>
```
Comment (shirkdog) [inline pulledpork.pl]: Need to keep -K

File: pulledpork.pl [hunk 2]
```diff
@@ -280,17 +278,10 @@ sub temp_cleanup {
 sub rule_extract {
     my (
         $rule_file, $temp_path, $Distro, $arch, $Snort,
-        $Sorules,   $ignore,    $docs,   $prefix
+        $Sorules,   $ignore,    $prefix
     ) = @_;
 
     #special case to bypass file operations when -nPT are specified
-    my $BypassTar = 0;
```
Comment (shirkdog) [inline pulledpork.pl]: Probably need this to remain, the rest should be able to be removed.

File: pulledpork.pl [hunk 3]
```diff
@@ -2115,7 +2089,6 @@ BEGIN
             if ($base_url =~ /[^labs]\.snort\.org/i) {
                 $prefix = "VRT-";
                 unless ($rule_file =~ /snortrules-snapshot-\d{4,6}\.tar\.gz/
-                    || $rule_file =~ /opensource\.gz/)
```
Comment (shirkdog) [inline pulledpork.pl]: Need to close the parentheses.

**Pull request merged**